docs/hosting/plaid.md

```diff
@@ -65,7 +70,7 @@ Go to [https://dashboard.plaid.com](https://dashboard.plaid.com) and register fo
 26. Click **Next**.
 27. Click **Start Security Practices Questionnaire**.
 28. For each question, select **Other - please see comments**, then write in the notes:  
```

Comment:
The questions in this specific  form changed a little since @PrplHaz4 wrote the step-by-step instructions but the spirit remains.  I simply answered the least strict of ways and got denied right away (by a bot, in all likelihood) then responded to their email with the explanation about personal use and got approved right away.